AustLII and NZLII.js

```diff
@@ -94,37 +89,124 @@ function doWeb(doc, url) {
 				articles.push(i);
 			}
 			ZU.processDocuments(articles, scrape);
-		});
-	} else {
+			return true;
+		}
+		);
+	}
+	else {
 		scrape(doc, url);
 	}
+	return false;
+}
+
+/*
+ * Convert full court names to standard abbreviations.
+ * If the name of the court is in the map, return the abbreviation
+ * else return the original full name (i.e. leave it unchanged)
+ * The Australian Guide to Legal Citation covers this in Rule 2.2.3
+ * legal Reports in Appendix A, but I've not been able to locate
+ * a copy with the appendix intact yet.
+*/
+function abbrevCourt(fullname) {
+	var courtMap = new Map();
+	courtMap.set('Federal Court of Australia', 'FCA');
+	courtMap.set('High Court of Australia', 'HCA');
+	courtMap.set('Family Court of Australia', 'FamCA');
+	courtMap.set('Australian Information Commissioner', 'AICmr');
```

Comment:
Keep this (and the `jMap` below) outside the function so we don't need to rebuild every time?